app/javascript/components/patient/Patient.js

```diff
@@ -720,6 +713,14 @@ class Patient extends React.Component {
                         <span className="risk-factor">Crew on Passenger or Cargo Flight</span>
                       </li>
                     )}
+                    {this.props.details.common_exposure_cohorts?.length > 0 && (
```

Comment:
Having this table with more than a few entries really messes with the look of the patient page (and takes up a lot of space).  any way we could collapse it on default, and only expand if the user wants

<img width="1753" alt="Screen Shot 2021-11-12 at 4 04 43 PM" src="https://user-images.githubusercontent.com/35042815/141534314-d6fa1091-a6f1-4f18-ba24-4b1d0ea74853.png">
?

Reply:
there should only be a max of 10 cohorts so hopefully that should solve the problem, if that isn't sufficient, it'll have to be done in a follow up release

Reply:
I think even with ten it's still a lot.  It wouldn't be that hard to add a `(show cohorts)` link next to the bullet point that toggles the table (similar to how we do it with lengthy notes)

Reply:
please discuss with PO team, I'm just following the JIRA ticket and mockups

Reply:
started a discussion here: https://mitrecorp.slack.com/archives/C029JM3PFC4/p1637187945098200

Reply:
will be discussed in the next feature review and the decision updated in a future version